packages/nc-gui/components/virtual-cell/QrCode.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import { useQRCode } from '@vueuse/integrations/useQRCode'
+
+const value = inject(CellValueInj)
```

Comment:
To avoid naming confusion this could be renamed to `cellValue` - then we wouldn't be accessing `value.value` 😅